web/src/components/FeedbackModal.tsx

```diff
@@ -28,80 +30,73 @@ const ModalContainer = styled.div`
 `
-const FeedbackContainer = styled.div`
+const ModalContent = styled.div`
   position: relative;
   display: flex;
+  flex-direction: column;
   background-color: ${props => props.theme.colors.backgroundColor};
 
   @media ${dimensions.smallViewport} {
     width: 100%;
     height: 100%;
-    align-items: flex-start;
+    align-items: center;
     justify-content: center;
   }
 `
+const Header = styled.div`
+  display: flex;
+  width: 360px;
+  padding: 20px;
+  flex-direction: row;
+  justify-content: space-between;
+  font-size: ${props => props.theme.fonts.subTitleFontSize};
+`
+
+const CloseButton = styled.button`
+  background-color: ${props => props.theme.colors.backgroundColor};
+  border: none;
+
+  & * {
+    font-size: 0.8em;
+    vertical-align: baseline;
+  }
+`
+
 type PropsType = {
   path?: string
   alias?: string
   cityCode: string
   language: string
-  route: RouteType
+  routeType: RouteType
   feedbackRating: FeedbackRatingType
-  closeFeedbackModal: () => void
+  closeModal: () => void
 }
 
-export type SendingStatusType = 'IDLE' | 'SUCCESS' | 'ERROR'
-type StateType = {
-  sendingStatus: SendingStatusType
-}
+const FeedbackModal = (props: PropsType): ReactElement => {
+  const { feedbackRating, closeModal, ...otherProps } = props
+  const { t } = useTranslation('feedback')
 
-export class FeedbackModal extends React.Component<PropsType, StateType> {
-  constructor(props: PropsType) {
-    super(props)
-    this.state = { sendingStatus: 'IDLE' }
-  }
-
-  handleSubmit = (sendingStatus: SendingStatusType): void => {
-    this.setState({
-      sendingStatus: sendingStatus
-    })
-  }
-
-  handleOverlayClick = (): void => {
-    this.setState({
-      sendingStatus: 'IDLE'
-    })
-    this.props.closeFeedbackModal()
-  }
-
-  renderContent(): React.ReactNode {
-    const { feedbackRating, ...otherProps } = this.props
-    const { sendingStatus } = this.state
-
-    if (['IDLE', 'ERROR'].includes(sendingStatus)) {
-      return (
-        <FeedbackBoxContainer
-          isPositiveRatingSelected={feedbackRating === POSITIVE_RATING}
-          onSubmit={this.handleSubmit}
-          sendingStatus={sendingStatus}
-          {...otherProps}
-        />
-      )
-    } else {
-      return <FeedbackThanksMessage closeFeedbackModal={this.props.closeFeedbackModal} />
-    }
-  }
-
-  render(): ReactNode {
-    return (
-      <FocusTrap>
-        <ModalContainer role='dialog' aria-modal>
-          <Overlay onClick={this.handleOverlayClick} />
-          <FeedbackContainer>{this.renderContent()}</FeedbackContainer>
-        </ModalContainer>
-      </FocusTrap>
-    )
-  }
+  return (
+    <FocusTrap>
+      <ModalContainer role='dialog' aria-modal>
+        <Overlay onClick={closeModal} />
+        <ModalContent>
+          <Header>
+            <div>{t('feedback')}</div>
+            <CloseButton aria-label={t('close')} onClick={closeModal}>
+              <FontAwesomeIcon icon={faTimes} />
+            </CloseButton>
```

Comment:
Why do we have a close button here and in `FeedbackContainer`?

Reply:
In FeedbackModal it is for the x in the Header, here it is for Closing the modal after sending the feedback.

Reply:
Is this new?

Reply:
No it is not.